README.md

```diff
@@ -1,19 +1,36 @@
-# go-slack-bot
-A super simple go slack bot
+# starter-go-bot
 
-Buiding the bot outside of docker requires [gb](https://getgb.io):
+## Overview
+A simple starting point for creating a Beep Boop hostable, Go based Slack bot.
 
-    go get github.com/constabulary/gb/...
-    gb build ./...
+Visit [Beep Boop](https://beepboophq.com/docs/article/overview) to get the scoop on the the Beep Boop hosting platform. The Slack API documentation can be found [here](https://api.slack.com/).
 
-Running locally:
+## Assumptions
+* You have already signed up with [Beep Boop](https://beepboophq.com) and have a local fork of this project.
+* You have sufficient rights in your Slack team to configure a bot and generate/access a Slack API token.
```

Comment:
Might want to mention here that you need Go 1.5 installed (tried with 1.4 first and failed gb build all).